setup.py

```diff
@@ -39,6 +39,7 @@
     'semver>=2.8.0',
     'nudatus>=0.0.3',
     'black>=18.9b0;python_version > "3.5"',
+    'Flask==1.0.3',
```

Comment:
Might be easier for debian/raspbian packaging (less patches needed) if the version is 1.0.2:
https://packages.ubuntu.com/disco/python3-flask
https://packages.debian.org/buster/python3-flask

Reply:
:+1:

Reply:
:+1: